include/BSMPT/bounce_solution/action_calculation.h

```diff
@@ -0,0 +1,821 @@
+   */
+  std::function<std::vector<double>(std::vector<double>)>
+      dV; // Potencial gradient
+  /**
+   * @brief Potential hessian of the class, completly numerical
+   *
+   * TODO: Calculate hessian from analytical gradient
+   */
+  std::function<std::vector<std::vector<double>>(std::vector<double>)> Hessian;
+  /**
+   * @brief First path given to class
+   *
+   */
+  std::vector<std::vector<double>> init_path; // Initial path given to class
+  /**
+   * @brief Current class path, can be changed by @ref PathDeformation
+   *
+   */
+  std::vector<std::vector<double>> path; // Initial path given to class
+  /**
+   * @brief We describe the tunneling path using a cubid spline.
+   *  The parameterization is the length along the spline.
+   *
+   */
+  cvspline::cvspline spline; // Constant velocity spline object
+  /**
+   * @brief Spline used to save \f$ \frac{dV}{dl} \f$
+   *
+   */
+  tk::spline rasterizedVdl; // RasterizeddVdl
+
+  /**
+   * @brief Default constructor
+   */
+  BounceActionInt();
+
+  /**
+   * @brief Construct a new Bounce Action Int object
+   *
+   * @param init_path is the initial path guess
+   * @param TrueVacuumIn is the true vacuum candidate of the potential
+   * @param FalseVacuumIn is the false vacuum
+   * @param V is the class potential
+   * @param dV is the class gradient
+   */
+  BounceActionInt(std::vector<std::vector<double>> init_path_In,
+                  std::vector<double> TrueVacuum_In,
+                  std::vector<double> FalseVacuum_In,
+                  std::function<double(std::vector<double>)> V_In,
+                  std::function<std::vector<double>(std::vector<double>)> dV_In,
+                  double T_In,
+                  int MaxPathIntegrations_in);
+
+  /**
+   * @brief Construct a new Bounce Action Int object
+   *
+   * @param init_path is the initial path guess
+   * @param TrueVacuumIn is the true vacuum candidate of the potential
+   * @param FalseVacuumIn is the false vacuum
+   * @param V is the class potential
+   */
+  BounceActionInt(std::vector<std::vector<double>> init_path_In,
+                  std::vector<double> TrueVacuum_In,
+                  std::vector<double> FalseVacuum_In,
+                  std::function<double(std::vector<double>)> &V_In,
+                  double T_In,
+                  int MaxPathIntegrations_in);
+
+  /**
+   * @brief Used set the path of the class.
+   *
+   * @param init_path Knots that are used to describe the path
+   */
+  void setPath(std::vector<std::vector<double>> init_path_In);
+
+  /**
+   * @brief Precalculates dVdl and creates a spline with the result.
+   * This is done to increase the runtime in large dimensional models
+   *
+   *  * @param l_start is the starting position produced in the
+   * backwardspropagation part
+   */
+  void rasterize_dVdl(double l_start = 0);
+
+  /**
+   * @brief Prints a vector
+   *
+   * @param vec Vector to be printed
+   */
+  static void PrintVector(std::vector<double> vec);
```

Comment:
Here you have three different naming conventions. Maybe pick one for the class and stick with it

Reply:
Fixed. Choose "PascalCase"